tests/kernel/version_header/README.rst

```diff
@@ -0,0 +1,29 @@
+.. _version_header:
+
+Version Header
+##############
+
+Overview
+********
+A simple program which add the version header to elf when built
```

Comment:
This should be a complete sentence:

    This program test that the version header is added to the 
    elf binary when the :option:`CONFIG_VERSION_HEADER`
    option is enabled.

Reply:
ok.